### DISCUSSION TITLE
installing inkscape and inkstitch

### DISCUSSION THREAD

**Discussion by Vbaxter212** [inkstitch/inkstitch#3379] [Q&A]
Can anyone please help.  I have installed inkstitch and Inkscape. onto my PC  However when I click on   Inkscape, it asks me to click on either repair, change or uninstall.  If I click on repair, nothing happens.  Or if  I click on change it still doesn't install.  I would be grateful if anyone could please provide the answer.  Many thanks Veronica

**Comment by bkmgit** [comment]
How did you install Inkscape and Ink/Stitch?  What operating system and operating system version are you using?

Does Inkscape launch if Ink/Stitch is not installed?

**Reply by bkmgit** [reply]
On Windows, anti-virus programs may disable inkstitch https://inkstitch.org/docs/install-windows/#troubleshoot

**Reply by Vbaxter212** [reply]
Thank you for your reply.  I have now tried to install Inkscape but it is greys out the https://inkscape.org and shows the website in a plain text format and the links don't work.  

Could you please advise how to resolve.  

Many thanks.  Veronica

**Reply by bkmgit** [reply]
May want to check the Inkscape communication channels. Assume you are using Windows, if so what version? How did you install Inkscape - which installer/executable did you download?
